docs/dsp-tools-create.md

```diff
@@ -67,44 +68,45 @@ As you can see in the example below, you can have more then one prefix too. In t
 ```
 
 ### "Project" object
-`"project": {"key": <value>", ...}`  
+`"project": {"key": "<value>", ...}`  
 
 Right after the "prefix" object the "project" object has to follow, which contains all resources and properties of the 
 ontology. The "project" object is the bread and butter of the ontology. All its important properties are specified therein. 
 
-As you saw in the complete ontology definition in the beginning, the project definitions `requires` ***exactly*** all 
-of the following datafields:
+As you saw in the complete ontology definition in the beginning, the project definitions requires all the following
+data fields:
 
 - shortcode
 - shortname
 - longname
 - keywords
 - ontologies
 
-Whereas the following fields are `optional` (if one or more of these fields are not
-used, it must be omitted):
+Whereas the following fields are optional (if one or more of these fields are not used, it must be omitted):
 
 - descriptions
 - lists
 - groups
 - users
 
-So a simple example definition of the "project" object could look like this:
+So, a simple example definition of the "project" object could look like this:
```

Comment:
Again, "So, ..." sounds more like an oral narrative than a software user manual

Reply:
Same here, I don't want to change the style - otherwise I would have to rewrite everything.